examples/all-clusters-app/esp32/main/include/LEDWidget.h

```diff
@@ -55,7 +55,7 @@ class LEDWidget
 #endif
 
 private:
-    int64_t mLastChangeTimeUS;
+    int64_t mLastChangeTimeMS;
```

Comment:
I don't understand this renaming.  This member is set to whatever its value used to be, plus `stateDurUS`.  Why is it being labeled as MS?

Reply:
Search-and-replace mistake; thanks for catching that. (This example doesn't use System::Clock.)